sr_armv7l/cmerrors_ctl.c

```diff
@@ -1,19 +1,18 @@
 /****************************************************************
  *								*
- * Copyright (c) 2001-2017 Fidelity National Information 	*
+ * Copyright (c) 2001-2017 Fidelity National Information	*
```

Comment:
A pre-existing copyright was removed in the following modules. I suspect that was unintentional.

sr_armv7l/cmerrors_ctl.c
sr_armv7l/cmierrors_ctl.c
sr_armv7l/gdeerrors_ctl.c
sr_armv7l/merrors_ansi.h
sr_armv7l/merrors_ctl.c

Reply:
You did not mention but I am assuming you mean the attribution of SteveJ? All of these routines are auto-generated by msg.m using the associated .msg file as input so all of them get the same copyright. If he didn't change the .msg file, he shouldn't have an attribution in any of these files. If he DID change the .msg file, the attribution there should suffice (IMHO).

Reply:
Yes that is what I meant. I don't think he changed the .msg file so the removal of that attrbiution is fine.